### PR TITLE
BN list update + selection process update

### PR DESCRIPTION
**Pull request by O1CDeif**
BN list:
- Taikocracy changed username to incandescence
- Added last batch of 20 osu! BNs and updated roles accordingly for Electoz and Gabe
- Removed -Sh1n1-
- Updated roles for Aloda (they can just taiko)
- Updated roles for Chromoxx (resigned from taiko)

Selection process:
- Added info about what's considered a mod post in modding v2 (towards the mod score count)
- Added info about the BN selection process

### CODE REVIEW

**Comment by MegaApplePi** [comment]
may conflict with #1011

**Comment by Xinnoh** [comment]
<The most recent change was on 01. February 2015 when the team was changed from the *Beatmap Appreciation Team* to the *Beatmap Nominators*.>
They were changed to the Beatmap Nomination Group back then, then they were changed to Beatmap Nominators more recently, need to update that.
Also, date should be formattd 2015-02-01

File: wiki/People/Beatmap_Nominators/en.md [hunk 1]
```diff
@@ -57,80 +65,99 @@ Also note: the game modes columns listed below indicates the game mode(s) in whi
...
+| [pinataman](https://osu.ppy.sh/u/1235485)         | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] |                          |
 | [Plaudible](https://osu.ppy.sh/u/7149815)         | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] |                          |
+| [Realazy](https://osu.ppy.sh/u/918297)            | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | French                   |
 | [Regraz](https://osu.ppy.sh/u/3076909)            | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | Chinese                  |
 | [Rivals_7](https://osu.ppy.sh/u/4610379)          | ![No][false] | ![No][false] | ![No][false] | ![Yes][true] | Indonesian               |
 | [sahuang](https://osu.ppy.sh/u/5318910)           | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | Chinese                  |
-| [Sinnoh](https://osu.ppy.sh/u/4236057)            | ![Yes][true] | ![No][false] | ![Yes][true] | ![No][false] |                         |
+| [Sinnoh](https://osu.ppy.sh/u/4236057)            | ![Yes][true] | ![No][false] | ![Yes][true] | ![No][false] |                          |
```
Comment (Xinnoh) [inline wiki/People/Beatmap_Nominators/en.md]: Some French

File: wiki/People/Beatmap_Nominators/en.md [hunk 2]
```diff
@@ -29,6 +29,7 @@ Modders that aim to apply to the Beatmap Nominators must fulfill the following c
 ### What Qualifies as a Mod Post
 + Upon signing up you will be asked to provide the total amount of Kudosu as well as the amount of mod posts within the last 3 months, sorted by month upon signing up. So if the announcement is posted in April the mod posts for January, February and March will be relevant. The relevant months will also be listed on the submission form.
 + A mod post qualifies as one if it has received a Kudosu and has been created within the time span listed on the submission form.
++ Since November 2017 beatmaps affected under Modding V2 got a reward system implemented, based on upvotes/downvotes. The "mod post" equivalent in this case is a beatmap discussion, in which at least one suggestion/problem got awarded with an upvote.
```
Comment (Xinnoh) [inline wiki/People/Beatmap_Nominators/en.md]: got a reward system -> had a reward system

Comment (Xinnoh) [reply wiki/People/Beatmap_Nominators/en.md]: implemented, based -> remove comma

File: wiki/People/Beatmap_Nominators/en.md [hunk 3]
```diff
@@ -29,6 +29,7 @@ Modders that aim to apply to the Beatmap Nominators must fulfill the following c
...
 + The gamemode (osu!, osu!mania, osu!catch, osu!taiko) that you modded beatmaps in does not matter as your activity in the modding community as a whole is being measured and not the activity in a certain game mode.
```
Comment (Xinnoh) [inline wiki/People/Beatmap_Nominators/en.md]: This is quite misleading. I chose to be less active in Mania after reading this and focused on fewer better mods, which was why I was rejected.

File: wiki/People/Beatmap_Nominators/en.md [hunk 4]
```diff
@@ -29,6 +29,7 @@ Modders that aim to apply to the Beatmap Nominators must fulfill the following c
...
 + The point in time at which the Kudosu has been awarded to you mod post does not matter, the only relevant information about a mod post is its post date and time in UTC+0 as well as the fact that it has been awarded Kudosu.
```
Comment (Xinnoh) [inline wiki/People/Beatmap_Nominators/en.md]: does not matter, the only -> Fullstop
in UTC+0 as well as the fact that it -> in UTC+0 and that it

File: wiki/People/Beatmap_Nominators/en.md [hunk 5]
```diff
@@ -48,6 +49,13 @@ Modders that aim to apply to the Beatmap Nominators must fulfill the following c
   + osu!: 1100
 + It is usually a good idea to keep their score well above these numbers as they can vary slightly depending on the amount of applicants.
 
+### How the new potential Beatmap Nominators are chosen
```
Comment (Xinnoh) [inline wiki/People/Beatmap_Nominators/en.md]: Remove the

File: wiki/People/Beatmap_Nominators/en.md [hunk 6]
```diff
@@ -48,6 +49,13 @@ Modders that aim to apply to the Beatmap Nominators must fulfill the following c
...
++ A doublecheck of the `Mod Score` and the `Kudosu Score` will be done to all applications, in order to find out if they reached the `Required Minimum Score`.
```
Comment (Xinnoh) [inline wiki/People/Beatmap_Nominators/en.md]: "The `Mod Score` and  `Kudosu Score` of all applicants will be double checked to confirm if they"
less wordy and fewer unused words

File: wiki/People/Beatmap_Nominators/en.md [hunk 7]
```diff
@@ -48,6 +49,13 @@ Modders that aim to apply to the Beatmap Nominators must fulfill the following c
...
++ Modders that acquired enough score will be evaluated by the correspondant QAT members of the game mode they applied for, taking into account the following criteria:
```
Comment (Xinnoh) [inline wiki/People/Beatmap_Nominators/en.md]: correspondant -> corresponding
one is a noun, one is an adjective

File: wiki/People/Beatmap_Nominators/en.md [hunk 8]
```diff
@@ -48,6 +49,13 @@ Modders that aim to apply to the Beatmap Nominators must fulfill the following c
...
+  + General attitude: Tone of delivery of the mods and other behavioural expectations from the [Code of Conduct](/wiki/Ranking_Criteria/Code_of_Conduct), as well as general behaviour in chat/forums.
+  + Modding abilities: Knowledge of the general [Ranking Criteria](/wiki/Ranking_Criteria) and specific criteria of each game mode. Additional abilities like Metadata, Timing, and some others will also be taken into consideration.
++ After balancing the pros and cons of each modder, a deliberation will be made whether the modder will become a Beatmap Nominator or not.
```
Comment (Xinnoh) [inline wiki/People/Beatmap_Nominators/en.md]: "a deliberation on whether the modder becomes a Beatmap Nominator will be made"
imo discussion would be easier to understand for non-English speakers

File: wiki/People/Beatmap_Nominators/en.md [hunk 9]
```diff
@@ -29,6 +29,7 @@ Modders that aim to apply to the Beatmap Nominators must fulfill the following c
 ### What Qualifies as a Mod Post
 + Upon signing up you will be asked to provide the total amount of Kudosu as well as the amount of mod posts within the last 3 months, sorted by month upon signing up. So if the announcement is posted in April the mod posts for January, February and March will be relevant. The relevant months will also be listed on the submission form.
```
Comment (Xinnoh) [inline wiki/People/Beatmap_Nominators/en.md]: Add commas after April and February
regarding feb's comma, it's an oxford comma which should be used.

File: wiki/People/Beatmap_Nominators/en.md [hunk 10]
```diff
@@ -48,6 +49,13 @@ Modders that aim to apply to the Beatmap Nominators must fulfill the following c
   + osu!: 1100
 + It is usually a good idea to keep their score well above these numbers as they can vary slightly depending on the amount of applicants.
```
Comment (Xinnoh) [inline wiki/People/Beatmap_Nominators/en.md]: their -> your
add  comma after as

File: wiki/People/Beatmap_Nominators/en.md [hunk 11]
```diff
@@ -48,6 +49,13 @@ Modders that aim to apply to the Beatmap Nominators must fulfill the following c
...
 
+### How the new potential Beatmap Nominators are chosen
++ A doublecheck of the `Mod Score` and the `Kudosu Score` will be done to all applications, in order to find out if they reached the `Required Minimum Score`.
++ Modders that acquired enough score will be evaluated by the correspondant QAT members of the game mode they applied for, taking into account the following criteria:
+  + General attitude: Tone of delivery of the mods and other behavioural expectations from the [Code of Conduct](/wiki/Ranking_Criteria/Code_of_Conduct), as well as general behaviour in chat/forums.
```
Comment (Xinnoh) [inline wiki/People/Beatmap_Nominators/en.md]: Tone of delivery of the mods -> Tone of delivery in mods

**Comment by cl8n** [comment]
could you change /u/ links to /users/? I didn't touch them in #1011 so you don't have to deal with any conflicts

File: wiki/People/Beatmap_Nominators/en.md [hunk 12]
```diff
@@ -57,80 +65,99 @@ Also note: the game modes columns listed below indicates the game mode(s) in whi
...
 | [Kawawa](https://osu.ppy.sh/u/4647754)            | ![No][false] | ![No][false] | ![No][false] | ![Yes][true] | Korean                   |
 | [Kencho](https://osu.ppy.sh/u/3178411)            | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | Cantonese, Chinese       |
 | [Kibbleru](https://osu.ppy.sh/u/3193504)          | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] |                          |
 | [Kin](https://osu.ppy.sh/u/480689)                | ![No][false] | ![Yes][true] | ![No][false] | ![No][false] | French                   |
 | [Kisses](https://osu.ppy.sh/u/4276114)            | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] |                          |
+| [Lanturn](https://osu.ppy.sh/u/1446665)           | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] |                          |
 | [Lasse](https://osu.ppy.sh/u/896613)              | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | German                   |
 | [Litharrale](https://osu.ppy.sh/u/3189995)        | ![No][false] | ![No][false] | ![No][false] | ![Yes][true] |                          |
```
Comment (Xinnoh) [inline wiki/People/Beatmap_Nominators/en.md]: lith is quit w

File: wiki/People/Beatmap_Nominators/en.md (hunk 4 above)
Comment (O1CDeif) [reply wiki/People/Beatmap_Nominators/en.md]: Too many "ands" in that sentence imo

**Comment by O1CDeif** [comment]
Updated after Sinnoh's revision. Also:
- Lith and Ozzy got removed
- Asherz got promoted to QAT
- Kuron-kun was added too
- Updated links to /users/

**Pull request merged**